zebra-chain/src/transaction/hash.rs

```diff
@@ -190,6 +197,17 @@ impl From<&Transaction> for WtxId {
     }
 }
 
+impl From<Arc<Transaction>> for WtxId {
+    /// Computes the witnessed transaction ID for a transaction.
+    ///
+    /// # Panics
+    ///
+    /// If passed a pre-v5 transaction.
+    fn from(transaction: Arc<Transaction>) -> Self {
+        transaction.as_ref().into()
+    }
+}
```

Comment:
👍

Reply:
It seems a little odd that a `From` impl should be sometimes panicking.